README.md

```diff
@@ -105,11 +112,17 @@ DEBUG_MODE=1 jest
 
 Run electron with `--no-sandbox`, set process env `JEST_ELECTRON_NO_SANDBOX=1`.
 
-
 ```bash
 JEST_ELECTRON_NO_SANDBOX=1 jest
 ```
 
+ - **additional startup arguments**
+
+Run electron with arbitrary arguments.
+
+```bash
+JEST_ELECTRON_STARTUP_ARGS='--disable-dev-shm-usage'
```

Comment:
JEST_ELECTRON_STARTUP_ARGS can contain `--no-sandbox`?

Reply:
Yes. It can contain any arbitrary argument.

Reply:
so we can remove the doc of `no-sandbox`, add a case in **additional startup arguments**